vanilla-framework/vanilla-framework
# Update mobile nav to use dropdowns for About and Patterns in docs

## Done

New styling for main nav and sidebar "component" nav at small viewports

## QA

- Pull code
- `cd docs`
- Run `./run`
- Open http://0.0.0.0:8104/
- Resize the viewport to small size
- Check the new dropdown-style toggles for the two navs and compare to the [design](https://github.com/vanilla-framework/vanilla-framework/issues/1948)

## Details
Fixes #1948
